docs/cli-mode.md

```diff
@@ -0,0 +1,281 @@
+# CLI Mode
+The CLI mode can be used for bulk resource management. 
+
+Usages:
+* Export all/selected resources from an Identity Server to a local directory.
```

Comment:
We keep saying `Identity Server` - Do we mean WSO2 IS specifically, or can it be any identity server? If it is WSO2 IS specifically, let's change this to `WSO2 IS`. Otherwise, we can say `from an identity server` (sentence case).

Reply:
This tool supports only our WSO2 IS. I have referred to as `an Identity Server` because this tool is intended to be used with multiple IS instances. So, yes, we can use `WSO2 IS` since we are referring to WSO2 IS specifically.